src/plugins/visualizations/public/wizard/type_selection/vis_help_text.tsx

```diff
@@ -42,7 +42,7 @@ interface VisHelpTextProps {
 export const VisHelpText = ({ name, title, description, highlightMsg }: VisHelpTextProps) => {
   return (
     <React.Fragment>
-      <EuiTitle size="s">
+      <EuiTitle size="m">
```

Comment:
Is this in a visualization? If so, let's not change - given its intentionally small, not sure if we should change either way. If you can point of screenshot, would help me give feedback

Reply:
this is a header in the select vis type modal, but it's not the major header. I agree we can leave it as small.
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/32652829/0ff39354-3c20-47e4-aeff-3e6ff98de730)